controllers/src/main/scala/cromwell/pipeline/controller/InputNodeController.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Route
+import cromwell.pipeline.datastorage.utils.auth.AccessTokenContent
+import cromwell.pipeline.service.InputNodeService
+import akka.http.scaladsl.server.Directives._
+import cromwell.pipeline.datastorage.dto.InputNode
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+
+import scala.concurrent.ExecutionContext
+import scala.util.{ Failure, Success }
+
+class InputNodeController(inputNodeService: InputNodeService)(implicit val ec: ExecutionContext) {
+  val route: AccessTokenContent => Route = _ =>
+    concat(
+      path("nodes") {
+        concat(
+          put {
+            concat(
+              entity(as[InputNode]) { request =>
+                onComplete(inputNodeService.updateNode(request)) {
+                  case Success(status) => complete(status)
+                  case Failure(e)      => complete(StatusCodes.InternalServerError, e.getMessage)
+                }
+              },
+              entity(as[List[InputNode]]) { request =>
```

Comment:
I think it could be POST method.

Reply:
In my opinion, POST is for insert and PUT is for update or else insert, so the last behavior is more suitable